src/api/content/service.ts

```diff
@@ -21,19 +22,33 @@ export interface ILanguage {
   messages: IMessageIdentifier
 }
 
-export interface ILanguageDataResponse {
-  data: ILanguage[]
-}
+export type ILanguageDataResponse = ILanguage[]
+
+export type CSVRow = { id: string; description: string } & Record<
+  string,
+  string
+>
 
 export async function getLanguages(
   application: string
 ): Promise<ILanguageDataResponse> {
-  return new Promise((resolve, reject) => {
-    readFile(
-      join('src/api/content/', `${application}/${application}.json`),
-      (err, data) => {
-        err ? reject(err) : resolve(JSON.parse(data.toString()))
-      }
-    )
+  const csvData = await readCSVToJSON<CSVRow[]>(
+    join('src/translations/', `${application}.csv`)
+  )
+  const languages = Object.keys(csvData[0]).filter(
+    (key) => !['id', 'description'].includes(key)
+  )
+
+  return languages.map((lang) => {
+    const messages: IMessageIdentifier = {}
+    csvData.forEach((row) => {
+      messages[row.id] = row[lang]
+    })
+
+    return {
+      lang,
+      displayName: lang,
```

Comment:
I think we use the display name for the "LanguageSelect" dropdowns. So they need to be you know proper labels